tests/main/nfs-support/task.yaml

```diff
@@ -0,0 +1,96 @@
+summary: Test that snaps still work when /home is a NFS mount
+details: |
+    Snapd now contains a feature where NFS-mounted /home (or any sub-directory)
+    initializes a workaround mode where all snaps gain minimal amount of network
+    permissions sufficient for NFS to operate.
+systems: [ubuntu-16.04-64]  # TODO: expand this list
```

Comment:
Do you plan to address this TODO in this PR?

Reply:
No, it's a lot of hairy stuff already. I'll address this with separate PRs.